### PR TITLE
arbwasm: add ProgramActivated event

### PR DESCRIPTION
**Pull request by tsahee**

### CODE REVIEW

**Review by rauljordan — APPROVED** [review]

File: src/precompiles/ArbWasm.sol [hunk 1]
```diff
@@ -54,6 +54,8 @@ interface ArbWasm {
     // @return gas cost paid per half kb uncompressed.
     function callScalar() external view returns (uint16 gas);
 
+    event ProgramActivated(address program, bytes32 codehash, uint16 version);
```
Comment (rachel-bousfield) [inline src/precompiles/ArbWasm.sol]: We should probably index this by codehash

**Review by rachel-bousfield — CHANGES_REQUESTED** [review]

File: src/precompiles/ArbWasm.sol (hunk 1 above)
Comment (tsahee) [reply src/precompiles/ArbWasm.sol]: done.
However - two questions:
1) should we really index by program? Anyone that knows program can find codehash and index with that
2) should we rename "program" to "address"?

Comment (rachel-bousfield) [reply src/precompiles/ArbWasm.sol]: I don't think we should index by program for exactly that reason.
I don't have strong feelings about renaming `program`.

**Review by rachel-bousfield — APPROVED** [review]
Looks great!

**Pull request merged**